crates/nu-cli/src/repl.rs

```diff
@@ -460,6 +481,121 @@ pub fn evaluate_repl(
     Ok(())
 }
 
+fn get_banner(engine_state: &mut EngineState, stack: &mut Stack) -> String {
+    let age = match eval_string_with_input(
+        engine_state,
+        stack,
+        None,
+        "(date now) - ('05/10/2019' | into datetime)",
+    ) {
+        Ok(Value::Duration { val, .. }) => format_duration(val),
+        _ => "".to_string(),
+    };
+
+    let banner = format!(
+        r#"{}     __  ,
+{} .--()°'.' {}Welcome to {}Nushell{}, 
+{}'|, . ,'   {}based on the {}nu{} language,
+{} !_-(_\    {}where all data is structured!
+
+Please join our {}Discord{} community at {}https://discord.gg/NtAbbGn{}
+Our {}GitHub{} repository is at {}https://github.com/nushell/nushell{}
+Our {}Documentation{} is located at {}http://nushell.sh{}
+{}Tweet{} us a {}@nu_shell{}
```

Comment:
Maybe either
```suggestion
{}Tweet{} us {}@nu_shell{}
```

or `Tweet us at ...`

Reply:
good catch. i'll update it.